src/audio/base_fw.c

```diff
@@ -71,7 +71,7 @@ static int basefw_config(uint32_t *data_offset, char *data)
 
 	/* TODO: add log support */
 	tuple = next_tuple(tuple);
-	set_tuple_uint32(tuple, IPC4_TRACE_LOG_BYTES_FW_CFG, 0);
+	set_tuple_uint32(tuple, IPC4_TRACE_LOG_BYTES_FW_CFG, 1024);
```

Comment:
Lets spell this out with a EXPERIMENTAL and DEPRECATED  Kconfig that sets the size. Kconfig help should make it obvious that this is tmp and will be removed without notice.

Reply:
@lgirdwood This mtrace method will live in parallel with DMA probes (both will be used). @pjdobrowolski is working on a cleaner version of this PR (and the backend will be put to Zephyr).

Reply:
ok, so wont this duplicate the Zephyr mailbox trace ? i.e. we will have 2 backend that write to shared SRAM ?

Reply:
@lgirdwood mtrace will replace the Zephyr mailbox trace. Linux kernel does not support the mailbox trace, but it does support mtrace.

Reply:
ok, but why cant Zephyr use the mtrace as its default BE for Intel DSP ?